GDJS/Runtime/runtimeobject.ts

```diff
@@ -1494,7 +1494,35 @@ namespace gdjs {
         this.getDrawableY() +
         this.getCenterY() -
         (otherObject.getDrawableY() + otherObject.getCenterY());
-      return (Math.atan2(-y, -x) * 180) / Math.PI;
+      return gdjs.toDegrees(Math.atan2(-y, -x));
+    }
+
+    /**
+     * Compute the X position when given an angle and distance relative to the starting object.
+     * This is also known as getting the cartesian coordinates of a 2D vector, using its polar coordinates.
+     * @param angle The angle, in degrees.
+     * @param distance The distance from the object, in pixels
+     */
+    getXFromAngleAndDistance(angle: float, distance: float): float {
```

Comment:
@4ian Here the based object function for the expression.

Reply:
Yep this one is fine because there is no confusion possible between the origin and the parameters :)